app/components/TableWrapper.tsx

```diff
@@ -69,8 +65,12 @@ export default function TableWrapper<T extends Record<string, unknown>>({
     ...initialQueryConfig,
   });
 
-  const query = useQueryBuilder(queryConfig, contextFilters);
-  const exportQuery = useExportQuery(queryConfig, contextFilters);
+  const visibleColumns = useMemo(
+    () => columns.filter((col) => !col.exportOnly),
+    [columns]
+  );
+  const query = useQueryBuilder(queryConfig, visibleColumns, contextFilters);
+  const exportQuery = useExportQuery(queryConfig, columns, contextFilters);
```

Comment:
just pointing out that we now pass columns into the query builder so that it can generate the query string form them, whereas before we were grabbing an array of column name strings from the `queryConfig`.